docs/plugins/repository-s3.asciidoc

```diff
@@ -145,6 +145,17 @@ settings belong in the `elasticsearch.yml` file.
     Whether retries should be throttled (i.e. should back off). Must be `true`
     or `false`. Defaults to `true`.
 
+`path_style_access`::
+
+   Whether to use the path style access pattern. If `true`, the path style access pattern will be used. If set to`false`,
```

Comment:
s/Whether to use the path style access pattern/Whether to force the use of the path style access pattern/